Appraisals

```diff
@@ -6,16 +6,20 @@ end
 
 appraise "rails-6.0" do
   gem "rails", "~> 6.0.0"
+  gem "tailwindcss-rails", "~> 0.5.1"
```

Comment:
@dixpac I've seen that `tailwindcss-rails` is now in version [2.0.3](https://github.com/rails/tailwindcss-rails/releases). Do you think we should bump the required version here (and below)?

Reply:
Yup, we should bump to >=1. You can also go with ~> 2 because I'm not sure that anyone uses version 1 anymore. 
But, for generators both 1 and 2 are the same.